zebra-chain/src/history_tree.rs

```diff
@@ -0,0 +1,245 @@
+//! History tree (Merkle mountain range) structure that contains information about
+//! the block history as specified in ZIP-221.
+
+use std::{
+    collections::{BTreeMap, HashSet},
+    io,
+    sync::Arc,
+};
+
+use thiserror::Error;
+
+use crate::{
+    block::{Block, ChainHistoryMmrRootHash, Height},
+    orchard,
+    parameters::{Network, NetworkUpgrade},
+    primitives::zcash_history::{Entry, Tree},
```

Comment:
There are many trees in our codebase now. Should we import this under an alias?

Reply:
Another alternative is only importing the module, and using `zcash_history::Tree` to refer to the type.

But using an alias is also fine.